optimade/validator/validator.py

```diff
@@ -670,7 +685,12 @@ def _construct_single_property_filters(
             # Otherwise, None values are not allowed for MUST's, and entire missing fields are not allowed
             raise ResponseError(msg)
 
+        using_fallback = False
         if prop_type == DataType.LIST:
+            if not test_value:
+                test_value = CONF.enum_fallback_values.get(endp).get(prop)
```

Comment:
Again, using the `get` method to retrieve something you know is there doesn't make much sense. At least, to me, the point of `get` is to be able to provide a fallback and avoid the `KeyError` exception. Not providing a fallback and then still relying on the retrieved value having a `get` method seems counter-intuitive.
If you _expect_ the key to be present, I'd rather see a `KeyError` exception than a `None doesn't have a get method` exception :)
```suggestion
                test_value = CONF.enum_fallback_values[endp][prop]
```

Reply:
But in this case we don't know it's there, this will get checked for every endpoint and field but only `structures->structure_features` has values

Reply:
Okay, so then I'll change the suggestion accordingly.

Reply:
Done

Reply:
But maybe you don't want to use `None` either? I don't know. I'm a bit lost in all this. It would really be great to have all these edge-cases tested.

Reply:
You saw how awkward the test case was for versions... this one is about 4 levels deeper than that :sweat_smile: 

I'm with you on the final suggestion though, will add it.

Reply:
The `None` will be caught in the next line and ignored after that, so its no big deal